.gitignore

```diff
@@ -1,3 +1,10 @@
 node_modules
 .DS_Store
 *.lock
+*.log
```

Comment:
Please remove .gitignore from this commit. The ignored files are not related to this project. Include them in your own global gitignore if needed.